x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_search.test.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mockKibanaValues } from '../../../../__mocks__';
+import '../../../__mocks__/engine_logic.mock';
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { EuiFieldSearch } from '@elastic/eui';
+
+import { AnalyticsSearch } from './';
+
+describe('AnalyticsSearch', () => {
+  const { navigateToUrl } = mockKibanaValues;
+  const preventDefault = jest.fn();
+
```

Comment:
You may want to add this. Your tests still pass without it, but if these tests are ever rearranged or new tests are added, it may start failing or giving false positives for `preventDefault`.

```suggestion

    beforeEach(() => {
      jest.clearAllMocks();
    })
    
```

Reply:
3f4d1a4